x-pack/plugins/cases/public/components/recent_cases/index.test.tsx

```diff
@@ -63,7 +63,7 @@ describe('RecentCases', () => {
         <RecentCases {...defaultProps} />
       </TestProviders>
     );
-    expect(getAllByTestId('case-details-link')).toHaveLength(7);
+    expect(getAllByTestId('case-details-link')).toHaveLength(8);
```

Comment:
On the mocks, I added to the array of cases a case with registered attachments to test the API responses. For that reason, I changed it from 7 to 8.